README.md

```diff
@@ -102,7 +102,6 @@ Invoke-Expression -Command (defang completion powershell | Out-String)
 
 The Defang CLI recognizes the following environment variables:
 
-- `COMPOSE_PROJECT_NAME` - The name of the project to use; overrides the name in the `compose.yaml` file
```

Comment:
This is still used, no? Why remove it?

Reply:
I was keeping support for the env var for backwards compatibility, but was hiding it from the docs in preference of the new `--project-name` flag. Happy to keep it around, but my impression was that the env var was a hack and the flag would now be the first-class way to configure this value.